README.md

```
@@ -240,461 +192,57 @@ For development, configure Claude Desktop to use your local build:
-
-#### Code Quality (Future)
+> **📖 For comprehensive development guides, testing strategies, and CI/CD information:**
+> - **[Testing Documentation](src/__test__/README.md)** - Unit tests, integration tests, coverage
+> - **[Docker Development](docs/docker.md)** - Container-based development workflows
+> - **[CI/CD Workflows](docs/workflows.md)** - GitHub Actions, automation, releases
 
-```bash
-npm run lint               # Lint code (needs ESLint setup)
-npm run format             # Format code (needs Prettier setup)
-```
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)